### ISSUE TITLE
linter line numbers off-by-one

### ISSUE THREAD

**Issue by davidmaxwaterman** [Polymer/polymer-linter#115]
I've noticed this a few times recently. The linter seems to report errors on lines that are one less than the line they are actually on (according to vim and grep, at least).

```
$ polymer --version
1.5.7
$ polymer lint src/*


          let data = await this._uploadFile(file, s3Filename));
                                                             ~
---->8----

src/my-file.html(323,61) error [parse-error] - Unexpected token )

$ grep -nr 'let data = await this._uploadFile(file, s3Filename))' src/my-file.html
324:          let data = await this._uploadFile(file, s3Filename));
```

linter reports it on line 323, but it is actually on line 324.

**Comment by rictic** [comment]
The polymer-linter uses zero-indexed lines and columns.

Maybe we should report the lines and columns as 1-indexed though? I notice that VSCode, Sublime, and Atom all report 1-based line and column indexes.

**Comment by usergenic** [comment]
Enhancement or bug?  Filing is enhancement.

**Comment by davidmaxwaterman** [comment]
> I notice that VSCode, Sublime, and Atom all report 1-based line and column indexes.

...and, more importantly, vi does too ;)

I wonder if a fix for this will affect plugins.

**Comment by rictic** [comment]
This won't affect plugins, will only affect printing to the console. Our internal representation will still be 0-based.
